backend/api/Services/MissionSchedulingService.cs

```diff
@@ -376,6 +388,14 @@ private async Task MoveInterruptedMissionsToQueue(IEnumerable<string> interrupte
                     return;
                 }
 
+                var unfinishedTasks = missionRun.Tasks
+                    .Where(t => !new List<Database.Models.TaskStatus>
+                        {Database.Models.TaskStatus.Successful, Database.Models.TaskStatus.Failed}
+                        .Contains(t.Status))
+                    .Select(t => new MissionTask(t)).ToList();
+
+                if (unfinishedTasks.Count == 0) continue;
+
                 var newMissionRun = new MissionRun
                 {
                     Name = missionRun.Name,
```

Comment:
I would change the name to make sure it shows that this mission only has the unfinished tasks, might not be obvious

Name = missionRun.Name + " unfinished tasks"

Reply:
Sounds good. I might need to have the default be in Norwegian, but I'm not sure. I suppose it depends on what we make the standard for the mission names. I can ask around.